ui/app/src/modules/Content/History/CompareVersionsDialog.tsx

```diff
@@ -152,7 +155,7 @@ function CompareVersionsDialog(props: CompareVersionsDialogProps) {
     shouldReject: ({ compareVersionsBranch, contentTypesBranch }) =>
       Boolean(compareVersionsBranch.error || contentTypesBranch.error),
     shouldRenew: ({ compareVersionsBranch, contentTypesBranch }, resource) =>
-      (compareVersionsBranch.isFetching || contentTypesBranch.isFetching) && resource.complete,
+      resource.complete,
```

Comment:
This will cause the component to suspend on every render. It was probably this way to avoid unintended "loadings". Let's discuss...

What's the piece of state that's changing that this is not catching and hence not renewing?

Reply:
Let's try it out and feel it.